_maps/map_files/generic/lavaland.dmm

```diff
@@ -2080,16 +2080,9 @@
 	},
 /area/mine/living_quarters)
 "en" = (
-/obj/structure/rack,
-/obj/item/weapon/storage/firstaid/regular,
-/obj/item/weapon/storage/firstaid/regular{
-	pixel_x = -3;
-	pixel_y = -3
-	},
-/turf/open/floor/plating{
-	baseturf = /turf/open/floor/plating/lava/smooth/lava_land_surface
-	},
-/area/mine/living_quarters)
+/obj/structure/lavaland_door,
```

Comment:
I forgot to mention that. sorry! The necropolis gate's position before the edit was actually on a tile to the left of the hallway, making the weather control machine visible and making it impossible to knock on the door if you were to its right but possible to if you were two tiles to its left. I did some variable manipulation and properly centered it in the hallway.